server/paymentProviders/thegivingblock/index.js

```diff
@@ -109,11 +109,14 @@ export const processOrder = async order => {
   await refresh(account);
 
   // create wallet address
-  const { depositAddress, pledgeId } = await createDepositAddress(account, {
+  const { depositAddress, pledgeId, errorMessage, meta } = await createDepositAddress(account, {
     organizationId: account.data.organizationId,
     pledgeAmount: order.data.customData.pledgeAmount,
     pledgeCurrency: order.data.customData.pledgeCurrency,
   });
+  if (errorMessage) {
+    throw new Error(`The Giving Block: ${errorMessage} ${meta.errorCode}`);
+  }
```

Comment:
Let's log the error and fail with a generic error message for now. Similar to Stripe:
https://github.com/opencollective/opencollective-api/blob/main/server/paymentProviders/stripe/creditcard.js#L18 

We'll later be able to have a list of errors that we actually want to share with the end users:
https://github.com/opencollective/opencollective-api/blob/main/server/paymentProviders/stripe/creditcard.js#L344

Reply:
Makes sense. 👍🏽 Corrected this. Now it shows as below.  

![image](https://user-images.githubusercontent.com/12435965/131521089-9000e5dc-b340-4216-8d09-b659009b8033.png)